webpack.common.config.js

```diff
@@ -25,6 +25,7 @@ var entry = {
     // JS
     'base-page': staticPath('js/pages/base-page.js'),
     'home-page': staticPath('js/pages/home-page.js'),
+    'landing-page': staticPath('js/pages/landing-page.js'),
```

Comment:
It's a little weird to have the homepage now be at `/home/`, with `/` called the "landing page" instead, especially since I [gather](https://github.com/caneruguz/osf.io/pull/17#issuecomment-175210892) that the Dashboard isn't going away, either. That's three things that seem to all occupy the same semantic space. How is the ... home page (the quick search implemented here) intended to relate to the dashboard?